src/posix/platform/netif.cpp

```diff
@@ -1213,6 +1251,12 @@ static void processNetifAddrEvent(otInstance *aInstance, struct nlmsghdr *aNetli
             addr6.sin6_family = AF_INET6;
             memcpy(&addr6.sin6_addr, RTA_DATA(rta), sizeof(addr6.sin6_addr));
 
+            // Ignore the required anycast address
```

Comment:
The code already explains itself for "ignoring required anycast addresses". We'd better explain why it should be ignored here.

Some ideas:

> Linux allows adding an IPv6 required anycast address to an interface, which blocks openthread deriving an address by SLAAC and will cause routing issues.
> Ignore the required anycast addresses here to allow OpenThread stack generate one when necessary, and Linux will prefer the non-required anycast address on the interface.

Reply:
Thanks！